lib/index.js

```diff
@@ -157,10 +159,14 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       // if `authenticationMethod` is `token-client-registration` then
       // register token client for it
       if(authenticationMethod === 'token-client-registration') {
-        // FIXME: get client ID from cookie
-        const clientId = 'FIXME';
+        // generate a client id
+        const clientId = await generateId({fixedLength: true});
+        // save it to the database
         await brAuthnToken.clients.set(
           {actor: null, email, account, clientId, authenticated: true});
+        // set a cookie in the response
+        const {name, maxAge} = cfg.cookies.clientId;
+        res.cookie(name, clientId, {maxAge, httpOnly: true});
```

Comment:
Actually, we should change this so it's easier to configure this cookie in other systems. We should just be passing the options in directly from the config. I'll make a note up where we set the config about this as well.